ocs_ci/deployment/helpers/hypershift_base.py

```diff
@@ -50,6 +50,27 @@ def get_hosted_cluster_names():
     return exec_cmd(cmd, shell=True).stdout.decode("utf-8").strip().split()
 
 
+def get_available_hosted_clusters_to_ocp_ver_dict():
+    """
+    Get available HyperShift hosted clusters with their versions
+
+    Returns:
+        dict: hosted clusters available with their versions. Example: {'cl-418-x': '4.18.7', 'cl-418-c': '4.19.0'}
+    """
+    logger.info("Getting HyperShift hosted clusters available")
+    cmd = (
+        "oc get hostedclusters -n clusters -o json | "
+        "jq -r '.items[] | "
+        'select(.metadata.annotations["hypershift.openshift.io/HasBeenAvailable"] == "true") | '
+        '"\\(.metadata.name)|\\(.status.version.history[0].version)"\''
+    )
```

Comment:
I'm thinking if it wouldn't be better to parse the output json directly in python. This is nice once-liner, but it might be little bit tricky to debug any issues there if there will be any change in the output or any other issue.

Reply:
tbh if I can continue with this one-liner for now and focus on enhancements later it will be more effective time management. Overloaded with Converged testing.